src/Microsoft.Health.Fhir.Shared.Api/Features/Resources/Bundle/BundleHandler.cs

```diff
@@ -888,6 +937,14 @@ private static OperationOutcome CreateOperationOutcome(OperationOutcome.IssueSev
             };
         }
 
+        private static string SanitizeString(string input)
+        {
+            return input
+                .Replace(Environment.NewLine, string.Empty, StringComparison.OrdinalIgnoreCase)
+                .Replace("\r", string.Empty, StringComparison.OrdinalIgnoreCase)
+                .Replace("\n", string.Empty, StringComparison.OrdinalIgnoreCase);
```

Comment:
I think all of these should be replaced with spaces. Sometimes there aren't spaces between the end of a line and the start of the next line so it would make for hard to read logs.